src/factories/reco/FarForwardNeutralsReconstruction_factory.h

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Sebouh Paul
+
+#pragma once
+
+#include "algorithms/reco/FarForwardNeutralsReconstruction.h"
+#include "algorithms/reco/FarForwardNeutralsReconstructionConfig.h"
+#include "services/algorithms_init/AlgorithmsInit_service.h"
+#include "extensions/jana/JOmniFactory.h"
+
+
+namespace eicrecon {
+
+  class FarForwardNeutralsReconstruction_factory : public JOmniFactory<FarForwardNeutralsReconstruction_factory,FarForwardNeutralsReconstructionConfig> {
+
+   using AlgoT = eicrecon::FarForwardNeutralsReconstruction;
+     private:
+         std::unique_ptr<AlgoT> m_algo;
+    PodioInput<edm4eic::Cluster> m_clusters_hcal_input {this};
+    PodioOutput<edm4eic::ReconstructedParticle> m_neutrons_output {this};
+    PodioOutput<edm4eic::ReconstructedParticle> m_gammas_output {this};
+    ParameterRef<std::vector<double>> m_n_scale_corr_coeff_hcal     {this, "n_scale_corr_coeff_hcal",          config().n_scale_corr_coeff_hcal};
+    ParameterRef<std::vector<double>> m_gamma_scale_corr_coeff_hcal     {this, "gamma_scale_corr_coeff_hcal",          config().gamma_scale_corr_coeff_hcal};
+    ParameterRef<double> m_rot_y     {this, "rot_y",          config().rot_y};
+    ParameterRef<double> m_gamma_zmax     {this, "gamma_zmax",          config().gamma_zmax};
+
+    ParameterRef<double> m_gamma_max_length     {this, "gamma_max_length",          config().gamma_max_length};
+    ParameterRef<double> m_gamma_max_width     {this, "gamma_max_width",          config().gamma_max_width};
```

Comment:
```suggestion
    ParameterRef<std::vector<double>> m_n_scale_corr_coeff_hcal     {this, "nScaleCorrCoeffHcal",          config().n_scale_corr_coeff_hcal};
    ParameterRef<std::vector<double>> m_gamma_scale_corr_coeff_hcal     {this, "gammaScaleCorrCoeffHcal",          config().gamma_scale_corr_coeff_hcal};
    ParameterRef<double> m_rot_y     {this, "rotY",          config().rot_y};
    ParameterRef<double> m_gamma_zmax     {this, "gammaZMax",          config().gamma_zmax};

    ParameterRef<double> m_gamma_max_length     {this, "gammaMaxLength",          config().gamma_max_length};
    ParameterRef<double> m_gamma_max_width     {this, "gammaMaxWidth",          config().gamma_max_width};
```
Config names exposed to JANA2 are camelCase.

Reply:
converted to camelCase